### ISSUE TITLE
Text track kind is misspelled

### ISSUE THREAD

**Issue by johnBartos** [google/shaka-player#808]
- What version of Shaka Player are you using?
2.1.1 

  - Can you reproduce the issue with our latest release version?
Yes

  - Can you reproduce the issue with the latest code from `master`?
Yes

- Are you using the demo app or your own custom app?
Both

  - If custom app, can you reproduce the issue using our demo app?
Yes

- What browser and OS are you using?
Chrome, FF, Edge, IE; MacOS

- What did you do?
Calling `getTextTracks`in the `trackschanged` event

  - What content did you load?
http://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd

    - If standard demo asset, which one?

    - If custom asset, what manifest and license server URIs?

  - How did you interact with the content, if at all?

- What did you expect to happen?
Tracks have a kind of `subtitles` or `captions`

- What actually happened?
Kind is `subtitle` or `caption`

I believe these enums are wrong: https://github.com/google/shaka-player/blob/01d64584224d4a9546cb0982b7047a6079f08c28/lib/util/manifest_parser_utils.js#L90-L93

According to the spec, valid kinds are `subtitles` or `captions` here.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track

**Comment by joeyparrish** [comment]
After careful review, we have determined that the typo is in the documentation for `shakaExtern.Stream` and `shakaExtern.Track`.  Those refer to "captions" and "subtitles", even though the constants used are "caption" and "subtitle".

I understand that it is a little confusing to have constants named very similarly to the `<track>` element attribute names, but the constants in `shakaExtern.Track` and `shakaExtern.Stream` are never applied to the `kind` attribute in `<track>`.

To summarize:
 - The Shaka Player API uses `subtitle` and `caption`.
 - DASH manifests use `subtitle` and `caption`.
 - HLS manifests use `subtitles`.
 - HTMLTrackElement uses `subtitles` and `captions`.

We will take this as a docs bug and correct the typos in the externs.  Thank you for the report!

**Comment by joeyparrish** [comment]
This fix was just released in v2.1.2.
